src/status_im2/contexts/quo_preview/settings/settings_item.cljs

```diff
@@ -0,0 +1,131 @@
+(ns status-im2.contexts.quo-preview.settings.settings-item
+  (:require
+    [quo2.core :as quo]
+    [react-native.core :as rn]
+    [status-im2.common.resources :as resources]
+    [reagent.core :as reagent]
+    [status-im2.contexts.quo-preview.preview :as preview]))
+
+(def descriptor
+  [{:label "Account name:"
```

Comment:
perhaps we can trim the fat of this descriptor thanks to the recent changes to preview screens code? :)